File: cppProgramming/Beginner/armstrong.cpp
#include<iostream>
using namespace std; 
int main()
{
	cout<<"enter the number"<<endl;
	int num;
	cin>>num;
	int rem,temp,sum=0;
	temp=num;
	while(num>0)
	{
		rem=num%10;
		sum=sum+rem*rem*rem;
		num=num/10;
	}
	if(sum==temp)
	{
		cout<<"armstrong number"<<endl;
	}
	else
	{
		cout<<"not an armstrong number" <<endl;
	}

}

File: pythonProgramming/armstrong.py
print("An armstrong number is a number whose sum of the cubes of individual digits will be equal to the number")
s = int(input("Enter the number to check if it is an armstrong number "))
num = s
sum = 0

while num > 0:
    rem = int(num%10)
    sum = sum+(rem*rem*rem)
    num = int(num/10)

if s == sum:
    print("It is an armstrong number")
else:
    print("It is not an armstrong number")

